packages/grafana-ui/src/components/PanelChrome/PanelDescription.tsx

```diff
@@ -34,9 +38,31 @@ export function PanelDescription({ description, className }: Props) {
   ) : null;
 }
 
-const getStyles = () => {
+const getStyles = (theme: GrafanaTheme2) => {
   return {
     description: css({
+      color: `${theme.colors.text.secondary}`,
+      backgroundColor: 'inherit',
+      cursor: 'auto',
+      border: 'none',
+      borderRadius: `${theme.shape.borderRadius()}`,
+      padding: `${theme.spacing(0, 1)}`,
+      height: '100%',
+      display: 'flex',
+      alignItems: 'center',
+      justifyContent: 'center',
+
+      '&:focus, &:focus-visible': {
+        ...getFocusStyles(theme),
+        zIndex: 1,
+      },
+      '&: focus:not(:focus-visible)': getMouseFocusStyles(theme),
+
+      '&:hover ': {
+        color: `${theme.colors.text.primary}`,
+        background: `${theme.colors.background.secondary}`,
+      },
+
```

Comment:
```suggestion
     
```

You can safely remove this, the `TitleItem` component is adding this style, I think this was leftover after resolving conflicts :)